### ISSUE TITLE
"Uncaught TypeError: Component is not a function" with HMR package

### ISSUE THREAD

**Issue by rj-david** [meteor/meteor#11715]
- [ ] The version of Meteor showing the problem.
Meteor 2.5

- [ ] The last version of Meteor where the problem did _not_ occur, if applicable.
Meteor 2.4

- [ ] The operating system you're running Meteor on.
Ubuntu  and Mac

- [ ] The expected behavior.
No error

- [ ] The actual behavior.
Error
```
Uncaught TypeError: Component is not a function
    at renderWithHooks (modules.js?hash=7fc9e69b0fdb1f0435a1db38acde5f36c079505a:21798)
    at updateForwardRef (modules.js?hash=7fc9e69b0fdb1f0435a1db38acde5f36c079505a:23857)
    at beginWork (modules.js?hash=7fc9e69b0fdb1f0435a1db38acde5f36c079505a:25911)
    at HTMLUnknownElement.callCallback (modules.js?hash=7fc9e69b0fdb1f0435a1db38acde5f36c079505a:10758)
    at Object.invokeGuardedCallbackDev (modules.js?hash=7fc9e69b0fdb1f0435a1db38acde5f36c079505a:10807)
    at invokeGuardedCallback (modules.js?hash=7fc9e69b0fdb1f0435a1db38acde5f36c079505a:10869)
    at beginWork$1 (modules.js?hash=7fc9e69b0fdb1f0435a1db38acde5f36c079505a:30777)
    at performUnitOfWork (modules.js?hash=7fc9e69b0fdb1f0435a1db38acde5f36c079505a:29589)
    at workLoopSync (modules.js?hash=7fc9e69b0fdb1f0435a1db38acde5f36c079505a:29520)
    at renderRootSync (modules.js?hash=7fc9e69b0fdb1f0435a1db38acde5f36c079505a:29483)
```

Error happens only when HMR is added to the project.

**Comment by filipenevola** [comment]
hi @zodern is this enough for you to reproduce the problem or do we still need a reproduction?

**Comment by nick-gudumac** [comment]
We are experiencing the same issue. Haven't been able to narrow it down yet.

**Comment by nick-gudumac** [comment]
This issue appears to be fixed now in Meteor 2.5.2. Where it crashed previously, now it's just doing a forced refresh.

**Comment by Grubba27** [comment]
From the comments and the missing activity in this issue, I think it has been solved or at least is not happening anymore, in the newer versions.
I will be closing this one for now. If for some reason is still happening I will happily open this one again
